javascript.ru.md

```diff
@@ -58,15 +58,15 @@ JavaScript CodeStyle
   * В именовании аббревиатур соблюдаем `CamelCase`. Например: `Json`, `Xml`.
 
 ##<a name="3"></a>Объявление переменных
-  * Все переменные объявляются с `var`.
+  * Все переменные должны быть объявлены перед использованием.
+  * Следует избегать как неявного использования глобальных переменных, так и глобальных переменных в принципе.
```

Comment:
It is different rules though.
To declare all variables means explicitly declare all variables with `var` statement. Even if undeclared variable will not become global.
To avoid global variables means to not use global variables unless it's absolutely necessary, like global namespace for modules system. Sometime developers wraps in closure `window` and `undefined`, hut I believe it's excessively.